### DISCUSSION TITLE
[@types/plotly.js] Marker color should accept a list

### DISCUSSION THREAD

**Discussion by marvson** [DefinitelyTyped/DefinitelyTyped#60789] [Issues with a @types package] [answered]
I'm not sure if it's a general behavior for all types of chart, but bar charts [can receive either a list of colors or a single color for each trace](https://plotly.com/javascript/bar-charts/#customizing-individual-bar-colors). Currently, it just accepts a single color.

[a comment by an automated account is not shown]

**Comment by brammitch** [comment, marked as answer]
[`marker.color`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20b6e0bcdd76bc66166a8574b83706ca8c6e16f7/types/plotly.js/index.d.ts#L1137) is of type [`Color`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20b6e0bcdd76bc66166a8574b83706ca8c6e16f7/types/plotly.js/index.d.ts#L1102-L1106), which can be an array.
